.drone.yml

```diff
@@ -0,0 +1,61 @@
+kind: pipeline
+name: linux-arm-1.0
+
+platform:
+  os: linux
+  arch: arm
+
+steps:
+- name: build
+  image: julia:1.0
+  commands:
+      - uname -a
+      - julia -e 'using Pkg; Pkg.clone(pwd()); Pkg.build("CMake"); Pkg.test("CMake"; coverage=true)'
```

Comment:
I think this and the other occurrences can simply be
```yaml
  - julia --project=. -e 'using Pkg; Pkg.build(); Pkg.test(coverage=true)'
```

Reply:
Ok! updated